.gitlab/pipeline/jobs/unit-php7.4-v11.yml

```diff
@@ -1,10 +1,10 @@
-unit-php7.4-v10:
+unit-php7.4-v11:
   extends: .default
   image: php:7.4
   stage: test
   needs:
     - build-composer-dependencies
     - php-lint-php7.4
   script:
-    - composer require --no-progress typo3/minimal:"^10.4"
+    - composer require --no-progress typo3/minimal:"^11.5"
```

Comment:
Should this not be `typo3/cms-core` instead ? (#530 for example) ? Not rebased correctly ? Or mabye slipped through there.